roles/lib_openshift/src/lib/base.py

```diff
@@ -9,6 +9,29 @@ class OpenShiftCLIError(Exception):
     pass
 
 
+def locate_oc_binary():
+    ''' Find and return oc binary file '''
+    # https://github.com/openshift/openshift-ansible/issues/3410
+    # oc can be in /usr/local/bin in some cases, but that may not
+    # be in $PATH due to ansible/sudo
+    paths = os.environ.get("PATH", os.defpath).split(os.pathsep) + ['/usr/local/bin', os.path.expanduser('~/bin')]
```

Comment:
Good point on sudo / root. This would need to move to happen before the change (which I'm guessing is unlikely) or use something like:


```python
os.path.expanduser('~' + os.getlogin()) + os.path.sep + 'bin'
```